### DISCUSSION TITLE
Clear feed history

### DISCUSSION THREAD

**Discussion by plwt** [community/community#4999] [Other Feature Feedback, Questions, & Ideas]
The one feature that I would most like the GitHub developers to consider is the ability for a user to clear their activity feeds ("All activity" and "Recent activity") through the Settings option in the profile drop-down menu.

Users may have moved on from a project and want to start from a clean state when they start a new project.  Another use case would be when a user is sharing their screen on a video call and may not want to broadcast what they are working on or following.

Thank you for giving me this opportunity to contribute an idea that I believe will be of use to many.

**Comment by bscadh** [comment]
Another use case I am in happens when I have ceased to follow/watch or have unstarred a repository. All past repository activities still remain on my activity feed. Having a way to delete them would be welcome.

Thank you!
